src/database/schema/ChannelListSchema.ts

```diff
@@ -517,9 +517,9 @@ class ChannelList implements BaseDbSchema {
       description: descriptionSystemMessage || firstMessage?.text || firstMessage?.message || '',
       unreadCount: data?.unreadCount ?? 0,
       channelType,
-      lastUpdatedAt: data?.last_message_at ?? data?.updated_at,
-      lastUpdatedBy: firstMessage?.user?.id,
-      createdAt: data?.created_at,
+      lastUpdatedAt: data?.last_message_at || data?.updated_at,
+      lastUpdatedBy: firstMessage?.user?.id || data?.created_by?.id,
+      createdAt: data?.created_at || data?.channel?.created_at,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are to provide alternative sources for `lastUpdatedAt`, `lastUpdatedBy`, and `createdAt` properties. This is a good practice as it ensures that these properties will always have a value, preventing potential runtime errors due to undefined values.
```diff
-      lastUpdatedAt: data?.last_message_at ?? data?.updated_at,
-      lastUpdatedBy: firstMessage?.user?.id,
-      createdAt: data?.created_at,
+      lastUpdatedAt: data?.last_message_at || data?.updated_at,
+      lastUpdatedBy: firstMessage?.user?.id || data?.created_by?.id,
+      createdAt: data?.created_at || data?.channel?.created_at,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->